source/extensions/omni.isaac.lab/test/sensors/test_camera.py

```diff
@@ -231,6 +231,56 @@ def test_multi_camera_init(self):
                 for im_data in cam.data.output.to_dict().values():
                     self.assertEqual(im_data.shape, (1, self.camera_cfg.height, self.camera_cfg.width))
 
+    def test_camera_init_intrinsic_matrix(self):
```

Comment:
does this also apply to the TiledCamera class? if so, should we include a test case in test_tiled_camera.py as well?

Reply:
I think that it would be useful if there was a test for it. I personally don't know the TiledCamera. Can you take that over? Should basically be identical to the test here.